lib/kvdb/indent.c

```diff
@@ -0,0 +1,158 @@
+
+#include "build_config.h"
+
+#include <string.h>
+
+#include <bsd/libutil.h>
+#include <bsd/string.h>
+
+#include <hse/hse.h>
+#include <hse/experimental.h>
+#include <hse/version.h>
+
+#include <hse/config/config.h>
+#include <hse/ikvdb/hse_gparams.h>
+#include <hse/ikvdb/ikvdb.h>
+#include <hse/ikvdb/kvdb_cparams.h>
+#include <hse/ikvdb/kvdb_ctxn.h>
+#include <hse/ikvdb/kvdb_home.h>
+#include <hse/ikvdb/kvdb_perfc.h>
+#include <hse/ikvdb/kvdb_rparams.h>
+#include <hse/ikvdb/kvs.h>
+#include <hse/ikvdb/kvs_cparams.h>
+#include <hse/ikvdb/kvs_rparams.h>
+#include <hse/ikvdb/limits.h>
+#include <hse/logging/logging.h>
+#include <hse/mpool/mpool.h>
+#include <hse/pidfile/pidfile.h>
+#include <hse/rest/headers.h>
+#include <hse/rest/params.h>
+#include <hse/rest/request.h>
+#include <hse/rest/response.h>
+#include <hse/rest/server.h>
+#include <hse/rest/status.h>
+#include <hse/util/err_ctx.h>
+#include <hse/util/event_counter.h>
+#include <hse/util/mutex.h>
+#include <hse/util/platform.h>
+#include <hse/util/vlb.h>
+#include <hse/util/compiler.h>
+
+#define HSE_KVS_PUT_MASK       1
```

Comment:
An example which includes end of line comments would be nice too.

Another example could be multiline macros.